src/App.js

```diff
@@ -1,27 +1,32 @@
 import React, { useState, useEffect } from "react";
 import "./style.css";
+import {FacebookShareButton,TwitterShareButton,WhatsappShareButton} from "react-share";
+let lang= "en";
 
 const App = () => {
   const [news, setNews] = useState([]);
-  var apiKey = `db01e884d61f4f908c66f1200c039fc4                                                                                                `;
-  const [searchQuery, setSearchQuery] = useState(["Trending"]);
+  var apiKey = `db01e884d61f4f908c66f1200c039fc4`;
+  const [searchQuery, setSearchQuery] = useState(["Trending World Wide"]);
   const [loading, setLoading] = useState(false);
-  //const [url, setUrl] = useState([`http://api.mediastack.com/v1/news?access_key=c35fdbfb0c5a2e16c400bbd2dbd3bca5&keywords=spiderman&sort=published_desc&limit=50&languages=en`])
   const [url, setUrl] = useState([
-    `http://newsapi.org/v2/everything?q=Trending&language=en&sortBy=publishedAt&apiKey=${apiKey}`,
+    `http://newsapi.org/v2/everything?q=Trending World Wide&language=${lang}&sortBy=publishedAt&apiKey=${apiKey}`,
```

Comment:
lang is not defined

Reply:
We can chat here??

Reply:
I had defined it
![image](https://user-images.githubusercontent.com/61278056/110124590-48ab4280-7de8-11eb-890d-f43fd91da700.png)

Reply:
Sorry didn't see the lang definition above. But it's better if it's in App().